### PR TITLE
New package: wrk

### PR DESCRIPTION
**Pull request by ifurther**
#4818

### CODE REVIEW

File: packages/wrk/Makefile.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
++	CFLAGS  += -I$(WITH_LUAJIT)/include/luajit-2.1
+ 	LDFLAGS += -L$(WITH_LUAJIT)/lib
+ else
+ 	CFLAGS  += -I$(ODIR)/include/luajit-2.1
+@@ -62,7 +62,7 @@
+ 
+ $(ODIR)/bytecode.o: src/wrk.lua
+ 	@echo LUAJIT $<
+-	@$(SHELL) -c 'PATH=obj/bin:$(PATH) luajit -b $(CURDIR)/$< $(CURDIR)/$@'
++	@$(SHELL) -c 'PATH=obj/bin:$(PATH):$(TERMUX_PREFIX)/bin luajit -b $(CURDIR)/$< $(CURDIR)/$@'
```
Comment (buttaface) [inline packages/wrk/Makefile.patch:17]: Should be `PREFIX`, not `TERMUX_PREFIX`.

Comment (buttaface) [reply packages/wrk/Makefile.patch:17]: No, wait, this runs on host, so `@TERMUX_PREFIX@`.

File: packages/wrk/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/wg/wrk
+TERMUX_PKG_DESCRIPTION="Modern HTTP benchmarking tool"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_VERSION=4.1.0
+TERMUX_PKG_SRCURL=https://github.com/wg/wrk/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6fa1020494de8c337913fd139d7aa1acb9a020de6f7eb9190753aa4b1e74271e
+TERMUX_PKG_DEPENDS="openssl, luajit"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_KEEP_STATIC_LIBRARIES=true
```
Comment (xeffyr) [inline packages/wrk/build.sh:9]: `TERMUX_PKG_KEEP_STATIC_LIBRARIES` is no-op. Remove it.

File: packages/wrk/Makefile.patch (hunk 1 above)
Comment (xeffyr) [reply packages/wrk/Makefile.patch:17]: So how x86_64 host will execute binary of different arch ?

**Pull request merged**